docs/source/jsonpath.rst

```diff
@@ -36,6 +40,8 @@ follows::
         # The string at beta has five chars
         $.beta.`len`: 5
 
-There are more JSONPath examples in :doc:`example`.
```

Comment:
Well, yes, but I ran out of energy (only made this change because I needed it for something else). Thus the links below to external sources. There are some examples in the tests too. If you've got the cycles to add something that's awesome, otherwise we can do it later.